modules/resource-group/README.md

```diff
@@ -0,0 +1,46 @@
+# resource-group creation module example
+
+This module is used to create a resource group.
+
+## Example Usage
+```
+
+module "resource_groups" {
+  source = "terraform-ibm-modules/cluster/ibm//modules/resource-group"
+
+
+  resource_group_name = var.name
+  tags    = var.tags
+}
+```
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
+## Inputs
+
+| Name                              | Description                                           | Type         | Default | Required |
+|-----------------------------------|-------------------------------------------------------|--------------|---------|----------|
+| name                              | Name of the resource group.                           | string       | n/a     | yes      |
+| tags                              | Tags associated with the resource group instance.     | list(string) | n/a     | no       |
```

Comment:
Tags associated with the resource group instance.  change to
Tags associated with the resource group.